exammode/templates/exammode/exam_end.html

```diff
@@ -0,0 +1,19 @@
+{% extends "exammode/base.html" %}
+{% load course %}
+{% load static %}
+
+{% block content %}
+<div class="text-center">
+    <p>
+        Please confirm that you have answered to all required questions before finsihing the exam. You won't be able to
```

Comment:
This text should be translated (`{% trans %}`).

Reply:
Done.